fdb-record-layer-core/src/main/proto/record_cursor.proto

```diff
@@ -133,6 +133,21 @@ message RecursiveCursorContinuation {
     optional bytes activeStateContinuation = 3;
 }
 
+message AggregateCursorContinuation {
+    optional bytes continuation = 1;
+    optional PartialAggregationResult partial_aggregation_results = 2;
+}
+
+message PartialAggregationResult {
+  optional bytes group_key = 1;
+  repeated AccumulatorState accumulator_states = 2;
+}
+
+message AccumulatorState {
+  optional string physical_operator_name = 1;
```

Comment:
Is the physical operator name actually used? From what I can tell, it grabs that information from the plan rather than the serialized `AccumulatorState`, which seems right anyway.

Reply:
It is not used now, unless we want to use it as a verify step (the accumulator's operator name = initialState.operatorName). I removed it.